test/index.js

```diff
@@ -183,45 +270,52 @@ describe('Repeat on cloned tree', function() {
 
 describe('URI', function() {
     it('to URI and back', function() {
-        var uri = new URI('/{domain:some}/path/to/something');
-        uri = new URI(uri);
-        uri.bind({domain: 'foo/bar'});
+        var uri = new URI('/{domain:some}/path/to/something', {}, true);
+        uri = new URI(uri, {domain: 'foo/bar'});
         deepEqual(uri.toString(), '/foo%2Fbar/path/to/something');
+        deepEqual(uri.expand().path, ['foo/bar','path','to','something']);
+    });
+
+    it('to URI and back, no pattern', function() {
+        var uri = new URI('/{domain:some}/path/to/something', {domain: 'foo'});
+        deepEqual(uri.toString(), '/%7Bdomain%3Asome%7D/path/to/something');
```

Comment:
Since this is explicitly not parsed & serialized as a pattern, encoding is fine to do. `encodeURIComponent` is quite conservative (so escapes more than strictly necessary), so we *could* add a less aggressive variant if we care.